config/prod/prow/jobs/config.yaml

```diff
@@ -21771,9 +22125,12 @@ periodics:
       args:
       - "./hack/release.sh"
       - "--dot-release"
-      - "--release-gcs knative-releases/eventing-rabbitmq"
-      - "--release-gcr gcr.io/knative-releases"
-      - "--github-token /etc/hub-token/token"
+      - "--release-gcs"
+      - "knative-releases/eventing-rabbitmq"
+      - "--release-gcr"
+      - "gcr.io/knative-releases"
+      - "--github-token"
+      - "/etc/hub-token/token"
       - "--branch release-0.25"
```

Comment:
This line `- "--branch release-0.25"` seems like not converted yet? Same on many other places

Reply:
Also there's a line such as this one: https://github.com/knative/test-infra/blob/357314fc91670759d6a7e64de3372129e8ebf23a/config/prod/prow/jobs/config.yaml#L214 (and others, just searched `args:` and found a few)
Should they all be converted?

Reply:
The following is an entirely different case (best to talk on separate thread). And in fact it looks proper. As a whole line is passed as a single argument to the other script.

https://github.com/knative/test-infra/blob/357314fc91670759d6a7e64de3372129e8ebf23a/config/prod/prow/jobs/config.yaml#L214